connect-examples/oauth/node/README.md

```diff
@@ -66,4 +66,7 @@ distributed under the License is distributed on an "AS IS" BASIS,
 WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
-```
\ No newline at end of file
+```
+
+## Feedback
+Leave a feedback for this sample app [here](https://delighted.com/t/DT6msOcY)!
```

Comment:
nit: Shouldn't it be `Leave feedback for this sample app here!`?

Reply:
Changing to `Rate this sample app here!`